RFC-by-stage/1-approved/STANDARDISED-ACCESSIBILITY-TESTING.md

```diff
@@ -0,0 +1,38 @@
+---
+Status: `Proposal` # Please do not change this.
+Implementer: # It will be changed upon merging and as it moves through the RFC stages
+---
+
+# Standardised manual accessibility testing
+
+## The issue to be solved
+
+Web accessibility is too subjective. Reading the WCAG success criteria is challenging and trying to agree on what constitutes a failure is hard, multiple people may have multiple views each slightly different. Normative failings (things which must pass) in WCAG becomes conflated with accessibility best practice and what people *feel* is poor accessibility. Agreeing on what constitutes a failure is the first step towards testing in a consistent way and having consistent results.
+
+Manual accessibility testing is aimed at conveying facts. Tests should strive to avoid personal preferences for identifying accessibility failures.
+
+This article forms the approach
+https://www.tpgi.com/heading-off-confusion-when-do-headings-fail-wcag/
+> WCAG techniques, such as H42: Using h1-h6 to identify headings and ARIA12: Using role=heading to identify headings, recommend that heading markup indicate the appropriate heading level for the content, but they don’t go so far as to define what’s “appropriate”—an issue that has been the subject of considerable discussion. **So although hierarchical heading structures reflect a best practice, skipping heading levels does not represent a WCAG failure.**
```

Comment:
> So although hierarchical heading structures reflect a best practice, skipping heading levels does not represent a WCAG failure

But https://www.w3.org/WAI/WCAG21/Techniques/general/G141.html does say:

> To facilitate navigation and understanding of overall document structure, authors should use headings that are properly nested (e.g., h1 followed by h2, h2 followed by h2 or h3, h3 followed by h3 or h4, etc.).

Reply:
But we don't want to confuse normative techniques (things which must pass) with informative techniques. Informative techniques help you apply the success criteria but aren't mandatory and are one identified way to address the success criteria.